version1/04 - Web services.md

```diff
@@ -1800,6 +1800,63 @@ Content-type: application/json
 - **servicePermissions**
   - the group service permissions
 
+### Get all objects explicit permissions for access groups by ID
+
+Retrieves a list of all objects with their respective explicit permissions for a certain access group
+
+**Request**
+
+```text
+GET /access-group/{id}/permissions?offset=0&limit=100
+Accept: application/json
+Authorization: Bearer {token}
+```
+
+###### Details
+
+- **id**
+  - the identifier of the access group to fetch
+- **offset (optional)**
+  - return objects from the specified offset
+- **limit (optional)**
+  - return n number of objects per page (maximum 100 and 10 by default)  
+
+**Response**
+
+200 OK
+Content-type: application/json
+{
+"results": [
+         {
+             "accessGroupId": int,
+             "explicitPermissions": [string],
+             "discriminator": string,
+             "objectId": int
+         },
+         {
+             "accessGroupId": int,
+             "explicitPermissions": [string],
+             "discriminator": string,
+             "listType": string
+         },
+         {
+             "accessGroupId": int,
+             "explicitPermissions": [string],
+             "discriminator": string,
+             "listType": string,
+             "code": string
+         }
+     ],
+     "hasMore": true
+}
+```
+
+###### Details
+
+- `results`
+  - contains the (potentially empty) objects with additional metadata and explicit permissions for the specified group
+  - at most 100 objets per page can be retrieved
```

Comment:
I think we must describe what values `discriminator` and `listtype` can take.

Reply:
We cam also mention what fields are expected for each discriminator type (i.e. when `objectId` is expected, when `listType` and when `listType` _and_ `code`)

Reply:
Additional information by the fields and the different types of discriminators and supported code lists and code list values was added. :)